src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static final String CUSTOM_REGEX = "//(.)\n(.*)";
+    public static final String REGEX = ",|:";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+
+        String[] texts = split(text);
+        int[] ints = toNumbers(texts);
+        return sum(ints);
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static String[] split(String text) {
+        Matcher matcher = Pattern.compile(CUSTOM_REGEX).matcher(text);
+        if (matcher.find()) {
+            String customDelimiter = matcher.group(1);
+            return matcher.group(2).split(customDelimiter);
+        }
+
+        String[] result = text.split(REGEX);
+        return result;
+    }
+
+    private static int[] toNumbers(String[] texts) {
+        int[] numbers = new int[texts.length];
+        for (int i = 0; i < texts.length; i++) {
+            numbers[i] = parse(texts[i]);
+        }
+
+        return numbers;
+    }
+
+    private static int parse(String text) {
+        int number = Integer.parseInt(text);
+        if (checkNegative(number)) {
+            throw new RuntimeException();
+        }
+
+        return number;
```

Comment:
```suggestion
        int number = Integer.parseInt(text);
        if (checkNegative(number)) {
            throw new RuntimeException();
        }

        return number;
```

- 규칙 3: 모든 원시값과 문자열을 포장한다.

해당 책임을 가지는 객체가 있으면 어떨까요?!
`PositiveNumber` 같은것으로요!

Reply:
리뷰어님 이 부분이 잘 이해가 가지 않습니다 ㅠㅠ